.github/workflows/_build.yml

```
@@ -34,7 +34,7 @@ on:
       conanfile:
         type: string
         required: false
-        default: conanfile.txt
+        default: conanfile.py
```

Do not change defaults, pass the proper value: https://github.com/czoido/flappy-frog/pull/1

```suggestion
        default: conanfile.txt
```